### DISCUSSION TITLE
YT music takes too time to load the music

### DISCUSSION THREAD

**Discussion by mounirare** [ReVanced/revanced-discussions#1113] [Question] [answered]
Hello there, 
Juste wondering if this a known issue.
I've been using the patched version of YT music for quite some time and for the last few days I've been noticing some strange behaviors. Either the music loads a long time before it starts, or it stops in the middle and loads for a few minutes. Add to that that playing the next title is no longer done automatically i have to click on next manually.
Has anyone ever encountered these problems ? Is there a way to solve these random loads ?

Thank you for your help.

**Comment by oSumAtrIX** [comment]
Already being fixed with https://github.com/ReVanced/revanced-patches/commit/cf3116a7583d09c25c798a85687a056f143656f0

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
@mounirare  If you have a computer and are familiar with command prompt (or can follow guides), you can patch with the CLI tools and use the dev release now.

Edit: dev release is now the normal release.  Can patch as usual with Manager app.

**Comment by mounirare** [comment]
Thanks for your quick response. 
I'll try the patch with CLI.

**Reply by LisoUseInAIKyrios** [reply]
Report back if the issue is fixed.

**Reply by mounirare** [reply]
Juste done patching the app ! After following guides to patch YT music with CLI the patched app is now working like a charm.
Thank you so much for your work and for your help.
